clang/lib/CIR/CodeGen/CIRGenBuiltin.cpp

```diff
@@ -49,6 +49,14 @@ RValue CIRGenFunction::emitBuiltinExpr(const GlobalDecl &gd, unsigned builtinID,
     }
   }
 
+  switch (builtinID) {
+  case Builtin::BI__builtin_complex: {
```

Comment:
II think the compiler will issue a warning if you don't have a default here, but the default should just break.